### PR TITLE
docs - config a server to access a dell ecs s3-compatible obj store

### PR DESCRIPTION
**Pull request by lisakowen**
include a dell ECS section in the s3 obj store config docs.  also use the term S3-compatible in some situations, and list out all three in others, both in content and topic titles.

the new dell ecs topic instructs the user to use the minio template.

### CODE REVIEW

File: docs/content/access_objstore.html.md.erb [hunk 1]
```diff
@@ -21,14 +21,14 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-PXF is installed with connectors to Azure Blob Storage, Azure Data Lake, Google Cloud Storage, MinIO, and S3 object stores.
+PXF is installed with connectors to Azure Blob Storage, Azure Data Lake, Google Cloud Storage, and AWS, MinIO, and Dell ECS S3-compatible object stores.
```
Comment (bradfordb-vmware) [inline docs/content/access_objstore.html.md.erb:24]:
```suggestion
PXF is installed with connectors to Azure Blob Storage, Azure Data Lake, Google Cloud Storage, AWS, MinIO, and Dell ECS S3-compatible object stores.
```

File: docs/content/intro_pxf.html.md.erb [hunk 2]
```diff
@@ -40,7 +40,7 @@ Your Greenplum Database deployment consists of a master host, a standby master h
 
 ## <a id="more"></a> About Connectors, Servers, and Profiles
 
-*Connector* is a generic term that encapsulates the implementation details required to read from or write to an external data store. PXF provides built-in connectors to Hadoop (HDFS, Hive, HBase), object stores (Azure, Google Cloud Storage, MinIO, S3), and SQL databases (via JDBC).
+*Connector* is a generic term that encapsulates the implementation details required to read from or write to an external data store. PXF provides built-in connectors to Hadoop (HDFS, Hive, HBase), object stores (Azure, Google Cloud Storage, and MinIO, AWS S3 and Dell ECS), and SQL databases (via JDBC).
```
Comment (bradfordb-vmware) [inline docs/content/intro_pxf.html.md.erb:43]:
```suggestion
*Connector* is a generic term that encapsulates the implementation details required to read from or write to an external data store. PXF provides built-in connectors to Hadoop (HDFS, Hive, HBase), object stores (Azure, Google Cloud Storage, MinIO, AWS S3 and Dell ECS), and SQL databases (via JDBC).
```

**Review by bradfordb-vmware — APPROVED** [review]

**Comment by lisakowen** [comment]
@denalex , i plan to merge this soon, please review if interested.

**Review by hpandeycodeit — APPROVED** [review]

**Comment by lisakowen** [comment]
i am going to merge this.  will address further comments in a separate commit/PR.

**Pull request merged**